src/tribol/mesh/CouplingScheme.hpp

```diff
@@ -571,9 +700,10 @@ class CouplingScheme
   bool m_isBinned; ///< True if binning has occured 
   bool m_isTied; ///< True if surfaces have been "tied" (Tied contact only)
 
-  InterfacePairs* m_interfacePairs; ///< List of interface pairs
+  ArrayT<InterfacePair> m_interface_pairs; ///< List of interface pairs
 
-  integer m_numActivePairs; ///< number of active interface pairs from InterfacePairs list
+  ArrayT<ContactPlane2D> m_contact_plane2d; ///< List of 2D contact planes
```

Comment:
It makes more sense for this to live on the `CouplingScheme` since the data is tied to each coupling scheme.  Before, the contact plane data would be rewritten as the coupling schemes are iterated through, which made it unclear whose coupling scheme the planes belong to.  I imagine this refactor is temporary; we should evaluate what is stored in the contact planes in the near term.